web/gds-user-ui/src/modules/dashboard/member/hook/useFetchMembers.tsx

```diff
@@ -0,0 +1,16 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { getMembersService } from '../service';
+import type { membersQuery, DirectoryType } from '../memberType';
+
+export function useFetchMembers(directory?: DirectoryType): membersQuery {
```

Comment:
I have a question. `directory` is optional here because we return `mainnet` by default if a `directory` is not provided?